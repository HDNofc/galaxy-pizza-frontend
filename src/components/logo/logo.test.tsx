import { MemoryRouter } from 'react-router-dom';
import renderWithThemeProvider from 'test-utils/render-with-theme-provider';
import Logo from './logo';

describe('<Logo>', () => {
  it('отображается корректно', () => {
    const { container } = renderWithThemeProvider(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('отображается корректно с пропсом `to`', () => {
    const { container } = renderWithThemeProvider(
      <MemoryRouter>
        <Logo to="/" />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
