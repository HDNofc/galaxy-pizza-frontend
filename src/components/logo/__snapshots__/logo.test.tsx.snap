// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo> отображается корректно 1`] = `
<div
  class="Logo-sc-u87jr6-0 jDjjYh"
>
  <svg
    class="LogoIcon-sc-u87jr6-2 cvzOuu"
    height="100%"
    width="24"
  >
    logo.svg
  </svg>
  <h1
    class="Title-sc-u87jr6-1 gOJvaA"
  >
    Galaxy Pizza
  </h1>
</div>
`;

exports[`<Logo> отображается корректно с пропсом \`to\` 1`] = `
<a
  class="Link-sc-16ffk5r-0 kUJqkA Logo-sc-u87jr6-0 jDjjYh"
  href="/"
>
  <svg
    class="LogoIcon-sc-u87jr6-2 cvzOuu"
    height="100%"
    width="24"
  >
    logo.svg
  </svg>
  <h1
    class="Title-sc-u87jr6-1 gOJvaA"
  >
    Galaxy Pizza
  </h1>
</a>
`;
